EarthAsDMAnalyzer/plugins/EarthAsDMAnalyzer.cc

```diff
@@ -377,6 +389,41 @@ void EarthAsDMAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup&
     bool isTrackerHighPtMuon = muon::isTrackerHighPtMuon(*muon, highestSumPt2Vertex);
     muon_isTrackerHighPtMuon_.push_back( isTrackerHighPtMuon);
 
+    // Match muon inner tracks with general track
+    bool hasMatchedGenTrack = 0;
+    float genTrackPt = 9999.; 
```

Comment:
this is better to be set to -9999, because it can be sometimes that the track pT is measured to be almost 10 TeV